csrc/scheduler/hopper_multi_matmul.cpp

```diff
@@ -435,17 +435,19 @@ void HopperMultipleMatmulScheduler::scheduleMmaResults() {
 }
 
 void HopperMultipleMatmulScheduler::scheduleEpilogue() {
-  // Load/cache the epilogue inputs if there are any.
-  auto& c_tvs = tensor_roles_.at(MatmulTensorRole::EPILOGUE_INPUT);
   std::vector<TensorView*> cached_tvs;
-  for (auto* c : c_tvs) {
-    cached_tvs.push_back(c->cacheAfter());
-  }
+  std::vector<TensorView*> c_tvs;
```

Comment:
I think this is unused. Its name is shadowed by a variable below that's local to the if scope.